easybuild/framework/easyblock.py

```diff
@@ -2348,6 +2348,13 @@ def post_install_step(self):
                 run_cmd(cmd, simple=True, log_ok=True, log_all=True)
 
         self.fix_shebang()
+        # GCC linker searches system /lib64 path before the LIBRARY_PATH paths
+        # However foreach path x in LIBRARY_PATH it searches x/../lib64 so we create that as a symlink to lib
```

Comment:
`foreach x` -> `for each <dir>/lib`
`x/../lib64` -> `<dir>/lib/../lib64`
`symlink to lib` -> `symlink to <dir>/lib/`
and please use `$LIBRARY_PATH`

Reply:
Slightly revised in https://github.com/easybuilders/easybuild-framework/pull/3401/commits/db3b5e656766b63bcbddfdefe6b0f10ce8cc7ad4